cypress/integration/articleFlows/postSidebarActions.spec.js

```diff
@@ -37,6 +43,8 @@ describe('Post sidebar actions', () => {
     cy.findByRole('link', { name: /^Share to Reddit$/i });
     cy.findByRole('link', { name: /^Share to Hacker News$/i });
     cy.findByRole('link', { name: /^Share to Facebook$/i });
+    // The navigator.share API should be available making this option visible
+    cy.findByRole('link', { name: /^Share Post via...$/i });
```

Comment:
We should probably have a separate test that checks it's not available when the stub isn't added - what do you think?